spring-integration-core/src/main/java/org/springframework/integration/graph/IntegrationGraphServer.java

```diff
@@ -66,7 +66,7 @@
  */
 public class IntegrationGraphServer implements ApplicationContextAware, ApplicationListener<ContextRefreshedEvent> {
 
-	private static final float GRAPH_VERSION = 1.2f;
+	private static final float GRAPH_VERSION = 1.3f;
```

Comment:
I wonder what have been changed in the `Graph` hierarchy to make a decision to bump the version...

Reply:
Just because there is a possibility of `"stats": null` which wasn't possible before.

Reply:
I still think it is not a framework matter. We definitely haven't changed our model.
It is out of our control how Jackson is going to be configured in the target application which indeed may affect the JSON is produced outside...